board/piksiv3/rootfs-overlay/etc/init.d/S86skylark1_upload_daemon

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
```

Comment:
You're right according to this https://www.quora.com/In-what-sequence-the-scripts-in-etc-init-d-directory-are-executed . I learned something new.